src/components/Auth/UsersCard/table/PassthroughMode.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Typography } from '@rmwc/typography';
+import React from 'react';
+
+import styles from './UsersTable.module.scss';
+
+export const PassthroughMode: React.FC = () => {
+  return (
+    <Typography
+      use="body2"
+      className={styles.noResultsWrapper}
+      theme="textSecondaryOnBackground"
+    >
+      <span>
+        No users persist when passthrough mode is enabled.{' '}
```

Comment:
Any thoughts on how to make this sentence a bit more clear? Auth emulator doesn't "persist" users -- they are either in memory or "exported". @markarndt

Reply:
What about:

> This table is unavailable because passthrough mode is enabled. Manage users in your custom Auth system instead.

Reply:
We should probably also update https://github.com/firebase/firebase-tools-ui/pull/654/files#diff-3f0b79d8c25faa664fe6f6a2f91a46da041f83608b6ca4a3e79fde9be88eabdcR33 if we change this language

Reply:
I'm fine merging as-is (but please use a different branch like `auth-new-features` instead). We can run a tech writer review round later. These strings are easy to change anyway.

Reply:
will do

Reply:
new base branch: https://github.com/firebase/firebase-tools-ui/tree/auth-new-features

Reply:
How about "...does not save user authentication credentials when passthrough is enabled...."?

Reply:
@markarndt Love that! I'd just say "does not save user information" though and that should cover what we're doing here. I'll draft a PR for the copy change.

Reply:
https://github.com/firebase/firebase-tools-ui/pull/659